src/commons/Config.ts

```diff
@@ -18,10 +19,18 @@ class Config {
     this.setConfig({});
   }
 
-  public setConfig(options: ConfigOptions) {
+  public async setConfig(options: ConfigOptions) {
     const {usePlatformColors = false, appScheme = 'light'} = options;
     this.usePlatformColors = usePlatformColors;
-    this.appScheme = appScheme;
+    this.appScheme = await AsyncStoragePackage?.getItem?.('appScheme') || appScheme;
```

Comment:
consider btw using a more unique id for the storage item, something like `rnuilib.appScheme`. 
So it won't conflict by any chance with other items people might set